src/lib/scene.js

```diff
@@ -3,12 +3,18 @@ import * as THREE from 'three';
 const scene = new THREE.Scene();
 const camera = new THREE.PerspectiveCamera(75, window.innerWidth / window.innerHeight, 0.1, 1000);
 const geometry = new THREE.BoxGeometry();
-const material = new THREE.MeshBasicMaterial({ color: 0x00ff00 });
+const material = new THREE.MeshStandardMaterial({ color: 0x00ff00 });
+material.metalness = 0.1;
 const cube = new THREE.Mesh(geometry, material);
 let renderer;
 scene.add(cube);
 camera.position.z = 5;
 
+// Add lighting
+const light = new THREE.DirectionalLight(0xffffff, 1);
+light.position.set(0, 0, 50);
+scene.add(light);
+
```

Comment:
Note this does not reflect the example - why did you choose this positioning?

There's a blend of directional and hemispheric with ground color, if you were concerned about accuracy.

Reply:
Same issues as metallic value, I don't know the actual values so it's difficult for me to set them properly.